ci-operator/jobs/openshift/cluster-api/openshift-cluster-api-release-4.0-postsubmits.yaml

```diff
@@ -0,0 +1,35 @@
+postsubmits:
+  openshift/cluster-api:
+  - agent: kubernetes
+    branches:
+    - ^release-4.0$
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      artifacts: images
+    name: branch-ci-openshift-cluster-api-release-4.0-images
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --promote
+        - --target=[images]
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-cluster-api-release-4.0.yaml
```

Comment:
This config is needed. Just copy `ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml` to `ci-operator/config/openshift/cluster-api/openshift-cluster-api-release-4.0.yaml`